src/style.css

```diff
@@ -1 +1,83 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+:root {
+  --main-color: #00acdc;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
it's forbidden to use anything except `box-sizing` in `*`

Reply:
https://css-tricks.com/reset-all-margins-padding/
what would be a better way to reset default margins and paddings? 
why is it forbidden? what consequences might it lead to?
thank you for pointing it out)

Reply:
The universal selector is one of the worst performers,